app/main.py

```diff
@@ -16,4 +16,27 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+
+def sort_variables(current_var: any) -> None:
+
+    if isinstance(current_var, (int, float, str, bool, tuple)):
+        sorted_variables["immutable"].append(current_var)
+    else:
+        sorted_variables["mutable"].append(current_var)
+
+
+sort_variables(lucky_number)
+sort_variables(pi)
+sort_variables(one_is_a_prime_number)
+sort_variables(name)
+sort_variables(my_favourite_films)
+sort_variables(profile_info)
+sort_variables(marks)
+sort_variables(collection_of_coins)
+
+print(sorted_variables)
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.